changelogs/unreleased/gh-10484-fix-space-DDL-memleak.md

```diff
@@ -0,0 +1,3 @@
+## bugfix/box
+
+- Fixed a memory leak on space DDL under certain conditions (gh-10484).
```

Comment:
Bad change log. Please be more specific.

Reply:
Changed to:

```
## bugfix/sql

- Fixed a memory leak on attempt to alter table view (gh-10484).
```